qt/aqt/utils.py

```diff
@@ -70,6 +120,107 @@ def openLink(link: str | QUrl) -> None:
         QDesktopServices.openUrl(QUrl(link))
 
 
+class MessageBox(QMessageBox):
+    def __init__(
+        self,
+        text: str,
+        callback: Callable[[int], None] | None = None,
+        parent: QWidget | None = None,
+        icon: QMessageBox.Icon = QMessageBox.Icon.NoIcon,
+        help: HelpPageArgument | None = None,
+        title: str = "Anki",
+        buttons: Sequence[str | QMessageBox.StandardButton] | None = None,
+        default_button: int = 0,
+        textFormat: Qt.TextFormat = Qt.TextFormat.PlainText,
+    ) -> None:
+        parent = parent or aqt.mw.app.activeWindow() or aqt.mw
+        super().__init__(parent)
+        self.setText(text)
+        self.setWindowTitle(title)
+        self.setWindowModality(Qt.WindowModality.WindowModal)
+        self.setIcon(icon)
+        self.setTextFormat(textFormat)
+        if buttons is None:
+            buttons = [QMessageBox.StandardButton.Ok]
+        for i, button in enumerate(buttons):
+            if isinstance(button, str):
+                b = self.addButton(button, QMessageBox.ButtonRole.ActionRole)
+            elif isinstance(button, QMessageBox.StandardButton):
+                b = self.addButton(button)
+            else:
+                continue
+            if callback is not None:
+                qconnect(b.clicked, partial(callback, i))
+            if i == default_button:
+                self.setDefaultButton(b)
+        if help is not None:
+            b = self.addButton(QMessageBox.StandardButton.Help)
+            qconnect(b.clicked, lambda: openHelp(help))
+        self.open()
+
+
+def ask_user(
+    text: str,
+    callback: Callable[[bool], None],
+    default_button: bool = True,
```

Comment:
How about `default_yes` or `default_no` to make it clearer?

Reply:
Done!